Mail/Views/New Message/ComposeMessageView.swift

```diff
@@ -118,9 +118,10 @@ struct ComposeMessageView: View {
         mailboxManager: MailboxManager,
         messageReply: MessageReply? = nil,
         attachments: [Attachable] = [],
-        textAttachment: TextAttachable? = nil
+        textAttachments: [TextAttachable] = []
```

Comment:
Compose message view can now accept an arbitrary collection of `TextAttachable` not just one.